File: legacy/src/main/java/br/com/legacy/managers/dtos/SearchFilter.java
package br.com.legacy.managers.dtos;

import java.io.Serializable;
import java.util.List;

import br.com.goin.component.model.event.api.response.ApiEvent;

public class SearchFilter implements Serializable {
    public int currentPage;
    public String totalPages;
    public String totalItems;
    public List<ApiEvent> events;
    public List<Club> clubs;
}

File: component_model_club/src/main/java/br/com/goin/component_model_club/model/SearchFilter.kt
package br.com.goin.component_model_club.model

import java.io.Serializable

import br.com.goin.component.model.event.api.response.ApiEvent

data class SearchFilter(var currentPage: Int = 0,
                        var totalPages: String? = null,
                        var totalItems: String? = null,
                        var events: List<ApiEvent>? = null,
                        var clubs: List<Club>? = null) : Serializable
